aicsimageio/aics_image.py

```diff
@@ -663,6 +664,23 @@ def metadata(self) -> Any:
         """
         return self.reader.metadata
 
+    @property
+    def ome_metadata(self) -> OME:
+        """
+        Returns
+        -------
+        metadata: OME
+            The original metadata transformed into the OME specfication.
+            This likely isn't a complete transformation but is guarenteed to
+            be a valid transformation.
+
+        Raises
+        ------
+        NotImplementedError
+            No XSLT file available for metadata transformation.
```

Comment:
I suggest changing this doc line to remove XSLT since it is an implementation detail itself.  It could just say "unsupported or unavailable metadata translation."